### DISCUSSION TITLE
瀏覽器操作問題

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS3073-C/Python-Community-GS3073-C#239] [Q&A] [answered]
今天(2022/2/15(二))老師上課時，有上網使用Microsoft Bing搜尋markdown來找尋markdown語法的中文說明。但，我發現我沒有辦法重現老師的結果。我一樣使用Microsoft Bing這個瀏覽器，也就是搜尋引擎來搜尋markdown，但沒有出現老師在上課時找到的「markdown語法說明」這個東西，我找到的東西全部都是英文。

**Comment by fatsoap** [comment]
在這邊
[Markdown 語法說明](https://markdown.tw/)

**Reply by enoch-lightning** [reply]
謝謝你的分享。不過，我還是好奇為什麼我的Microsoft Bing找不到老師上課講的，也就是你提供的Markdown語法說明。

**Reply by enoch-lightning** [reply]
![image](https://user-images.githubusercontent.com/86505454/154239077-d686c97a-1701-49ff-88d1-38443f98066b.png)
![image](https://user-images.githubusercontent.com/86505454/154239090-59701570-8f76-4da5-a8a2-fd32a1eacb5c.png)
可以請問你上面這個markdown表情符號列表，是你自己手工一列一列打出來的嗎？看起來好厲害。

**Reply by enoch-lightning** [reply]
我也想請問你要怎麼看你這個網頁，你的說明說這是你的個人網頁(personal web)
![image](https://user-images.githubusercontent.com/86505454/154239900-3380cae8-043f-4dd6-87ec-1836ce972662.png)
謝謝

**Reply by fatsoap** [reply]
> ![image](https://user-images.githubusercontent.com/86505454/154239077-d686c97a-1701-49ff-88d1-38443f98066b.png)
> 
> ![image](https://user-images.githubusercontent.com/86505454/154239090-59701570-8f76-4da5-a8a2-fd32a1eacb5c.png)
> 
> 可以請問你上面這個markdown表情符號列表，是你自己手工一列一列打出來的嗎？看起來好厲害。

這是 fork 別人的 emoji cheat sheet 

原本的網址在這邊[emoji](https://gist.github.com/rxaviers/7360908)

**Reply by enoch-lightning** [reply]
謝謝

**Comment by fatsoap** [comment]
另外今天老師上課用的 VScode有內建預覽 markdown 的功能，在開啟 .md 檔案後右上角會出現下圖的標示，按一下就可以預覽
![image](https://user-images.githubusercontent.com/45852534/154103133-e41e0a0d-4c69-4281-bb89-283b1f541f7f.png)

**Reply by enoch-lightning** [reply]
我來試試看，謝謝你的補充。

**Comment by enoch-lightning** [comment, marked as answer]
我忘記昨天2022/2/15（二）老師說看到什麼就代表markdown了。

**Reply by fatsoap** [reply]
副檔名是 .md

**Reply by enoch-lightning** [reply]
謝謝你。
